src/components/MenuBar.vue

```diff
@@ -70,13 +70,21 @@ export type MenuItemType = MenuItemData["type"];
 const store = useStore();
 const $q = useQuasar();
 const currentVersion = ref("");
-const altPorts = ref<number[]>([]);
 
-store.dispatch("GET_ALT_PORT_INFOS").then(
-  (altPortInfo) =>
-    // {[engineId]: {from: number, to: number}} -> to: number[]
-    (altPorts.value = Object.values(altPortInfo).map(({ to }) => to))
-);
+// デフォルトエンジンの代替先ポート
+const defaultEngineAltPortTo = computed<number | undefined>(() => {
+  const altPortInfos = store.state.altPortInfos;
+
+  // ref: https://github.com/VOICEVOX/voicevox/blob/32940eab36f4f729dd0390dca98f18656240d60d/src/views/EditorHome.vue#L522-L528
+  const defaultEngineInfo = Object.values(store.state.engineInfos).find(
+    (engine) => engine.type === "default"
+  );
+  if (defaultEngineInfo == null) return undefined;
+
+  // <defaultEngineId>: { from: number, to: number } -> to (代替先ポート)
+  return altPortInfos[defaultEngineInfo.uuid]?.to;
```

Comment:
altPortInfosになかった場合は正常にundefinedを返す、ということを良い感じに伝えられるコードにするのはどうでしょう。こんな感じ･･･？

```suggestion
  if (defaultEngineInfo.uuid in altPortInfos) {
    return altPortInfos[defaultEngineInfo.uuid].to;
  }
  else {
    return undefined;
  }
```